multiworld/envs/mujoco/__init__.py

```diff
@@ -277,6 +277,7 @@ def register_custom_envs():
             hand_high=(0.0, 0.65, 0.2),
             action_scale=0.02,
             hide_goal_markers=True,
+            num_goals_presampled=1000,
```

Comment:
In general, it'll be good practice to update the version (and git commit hash) whenever you make changes. I think you're the only one using this env, so I think it's fine.